### PR TITLE
fix(TopBar): use the prop className

### LINKED ISSUE #239: Minor Styling bug (TopBar): Cisco logo is not being pushed to the right
Screen Shot 2019-05-13 at 9 45 20 AM

Just a bit of left margin on the logo should fix this issue.

### LINKED ISSUE #260: [Topbar] className not used, `row` class potentially missing on the container
**⚠️ Update: this issue and the related PR (https://github.com/momentum-design/momentum-ui/pull/261) is now only focused on the usage of the `className` props int he React `TopBar` component. See https://github.com/momentum-design/momentum-ui/issues/239 for the look and feel of the component.**

I did **not** try reporting this issue to the "Momentum UI Q&A" space in Webex Teams before creating a new GitHub issue, don't hate me.

I was trying to understand why the react <TopBar /> component does not look as nice as the [pure HTML/CSS example from the documentation](https://momentum.design/components/top-bar/code), and at the same time I found out that the prop `className` is not used.

## Issue Type
All of them:
* Bug Report
* Feature Request
* Support Request

## Component or Pattern Affected
TopBar

## Expected Behavior

Screenshot_2019-06-07 Momentum UI React Starter(1)

## Current Behavior

Screenshot_2019-06-07 Momentum UI React Starter

## Possible Solution
To make the top bar look like the screenshot in the "Expected Behavior", one needs to add the class `row` to `<div class="md-top-bar__container">` but [there is not way to do it with the current React component](https://github.com/momentum-design/momentum-ui/blob/700936bb792f2a7844e3c8536d68543110d6b59d/react/src/lib/Topbar/index.js#L98). I tried using [the `className` property](https://github.com/momentum-design/momentum-ui/blob/700936bb792f2a7844e3c8536d68543110d6b59d/react/src/lib/Topbar/index.js#L114-L115), only to discover that [it isn't used at all](https://github.com/momentum-design/momentum-ui/blob/700936bb792f2a7844e3c8536d68543110d6b59d/react/src/lib/Topbar/index.js#L30-L38).

## Steps to Reproduce (for bugs)
Use `<TopBar />`.

## Your Environment
* Library: React
* Version used: 17.3.4
* Browser Name and version: Firefox 69
* Operating System and version: macOS 10.14.5
* Link to your project: 🤡

### PR DESCRIPTION
**Pull request by tbassetto**
## Description
See #260 

The `className` change if ok I think, but adding `row` may be **very naive**. Maybe it works better only on "sites" and not on "apps". But are apps supposed to use this TopBar component anyway? Also, I did not update the other TopBar components (like the Angular one).

## Related Issue
fixes #260 & fixes #239 

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: react/src/lib/Topbar/index.js [hunk 1]
```diff
@@ -89,13 +90,14 @@ class Topbar extends React.Component {
       <div
         className={`${mdTopBarClass}` +
         `${(fixed && ` ${mdTopBarClass}--fixed`) || ''}` +
+        `${className || ''}` +
```
Comment (bfbiggs) [inline react/src/lib/Topbar/index.js]: Nice catch - just need to add the formatting for the space: 
```
`${(className && ` ${className}`) || ''}`
```

**Comment by bfbiggs** [comment]
With the className change - don't forget to run ```yarn test -u``` in the react directory to update the Topbar snapshot.

**Comment by tbassetto** [comment]
Thanks for the feedback, I'll fix this sometimes PR during the week.

**Comment by tbassetto** [comment]
Updated the PR to focus on `className` only.

**Comment by tbassetto** [comment]
Forgot to update the commit message… Will do it asap.

**Pull request merged**